### PR TITLE
Add support for Django 4.2

### PR DESCRIPTION
**Pull request by Natim**
Slippers 0.5 locks down Django 4.2 update.
Let's add it to the scope of the library.

```
Because slippers (0.5.0) depends on Django (>=3.2,<4.2)
```

### CODE REVIEW

**Comment by adamchainz** [comment]
It doesn't *prevent* the update after #32 (merged but unreleased).

File: poetry.lock [hunk 1]
```diff
@@ -1,10 +1,16 @@
+# This file is automatically @generated by Poetry 1.4.2 and should not be changed by hand.
```
Comment (adamchainz) [inline poetry.lock:1]: Does anything else need to change given you're upgrading Poetry?

(If this was my project I would request to split such changes.)

Comment (Natim) [reply poetry.lock:1]: The documentation [states `poetry>=1.0`](https://github.com/mixxorz/slippers/blob/main/pyproject.toml#L45) 

I don't think it has any impact on the project to use the last released version of Poetry no.

**Comment by Natim** [comment]
@mixxorz any thought about this PR?

**Comment by mixxorz** [comment]
Hi. I think we can close this. I'll release 0.6.0 in a little bit which should allow you to install on any future Django version.

**Pull request closed without merging**